src/OpenWeatherMapApi/OpenWeatherMapClient.cs

```diff
@@ -14,12 +16,12 @@ public class OpenWeatherMapClient
 		private readonly HttpClient _client;
 
 		/// <summary>
-		/// Initializes a client to retireve data from OpenWeaterMap
+		/// Initializes a client to retrieve data from OpenWeaterMap
 		/// </summary>
-		/// <param name="apiKey">Your OpenWeatherMap Api Key</param>
-		public OpenWeatherMapClient(string apiKey, HttpClient client = null)
+		/// <param name="apiKey">Your OpenWeatherMap Api Key in the struct OpenWeatherMapAPIKey.</param>
+		public OpenWeatherMapClient(OpenWeatherMapAPIKey apiKey, HttpClient client = null)
```

Comment:
Doesn't maintain compatibility with where its currently used. Major version incrementation or add a constructor taking the api key as a string still.